vcr/credential/util.go

```diff
@@ -140,3 +141,37 @@ func AutoCorrectSelfAttestedCredential(credential vc.VerifiableCredential, reque
 	}
 	return credential
 }
+
+// FilterOnDIDMethod filters the credentials based on the DID method of the issuer and credentialSubject.
+// credentials are only removed if there's a false match. Nil fields are ignored.
+func FilterOnDIDMethod(credentials []vc.VerifiableCredential, didMethods []string) []vc.VerifiableCredential {
+	var result []vc.VerifiableCredential
+outer:
+	for _, credential := range credentials {
+		// check issuer
+		issuerDID, err := did.ParseDID(credential.Issuer.String())
```

Comment:
The `err != nil` case on both `did.ParseDID(Issuer | Subject)` is ignored, what is the reasoning for this?